source/_integrations/no_ip.markdown

```diff
@@ -4,41 +4,18 @@ description: Keep your computer registered with the NO-IP.com dynamic DNS.
 ha_category:
   - Network
 ha_iot_class: Cloud Polling
+ha_config_flow: true
 ha_release: 0.57
 ha_domain: no_ip
 ha_integration_type: integration
 ---
 
-With the `no_ip` integration you can keep your current IP address in sync with your [NO-IP.com](https://www.noip.com)  hostname or domain.  
+With the No-IP integration, you can keep your current IP address in sync with your [No-IP.com](https://www.noip.com) hostname or domain.
 
-Note that it does not confirm your hostname (as is required periodically for free domain names); you will still need to do that manually.
+Please note that the integration does not automatically confirm your hostname (which is required periodically for free domain names). You will still need to do that manually.
 
-To use the integration in your installation, add the following to your `configuration.yaml` file:
+Additionally, you have the flexibility to customize the timeout duration (default: 10 seconds) and the update interval (default: 5 minutes) according to your preferences.
```

Comment:
This is not allowed to be provided by integrations anymore.

Reply:
Hello, do you mean the timeout or the update interval

Reply:
both, I've left reviews on the PR